core/src/test/java/com/sequenceiq/cloudbreak/reactor/handler/cluster/upgrade/ccm/RemoveAgentHandlerTest.java

```diff
@@ -51,31 +51,30 @@ void selector() {
 
     @Test
     void doAccept() throws CloudbreakOrchestratorException {
```

Comment:
Perhaps also verify that `upgradeCcmService.removeAgentFailed()` is never invoked in this case.

Reply:
removeAgentFailed is called in this case too

Reply:
Still unresolved.